core/src/main/resources/hudson/util/HudsonIsLoading/index.jelly

```diff
@@ -40,23 +40,19 @@ THE SOFTWARE.
             <!-- mobile friendly layout -->
             <meta name="viewport" content="width=device-width, initial-scale=1" />
             <link rel="stylesheet" href="${resURL}/jsbundles/simple-page.css" type="text/css" />
-            <link rel="stylesheet" href="${resURL}/css/loading.css" type="text/css" />
         </head>
         <body>
             <div class="simple-page" role="main">
-                <div class="modal signup">
-                    <div class="signupIntroDefault">
-                        <div class="logo">
-                          <img src="${imagesURL}/svgs/logo.svg" alt="Jenkins logo" />
-                        </div>
-                        <h1 class="loading">
-                            ${%Please wait while Jenkins is getting ready to work}
-                            <span>.</span><span>.</span><span>.</span>
-                        </h1>
-                        <p class="restarting">
-                            ${%Your browser will reload automatically when Jenkins is ready.}
-                        </p>
+                <div class="app-jenkins-booting">
+                    <div class="logo">
+                      <img src="${imagesURL}/svgs/logo.svg" alt="Jenkins logo" />
                     </div>
+                    <h1 class="loading">
+                        <l:spinner text="${%Please wait while Jenkins is getting ready to work}" />
+                    </h1>
+                    <p class="restarting">
+                        ${%Your browser will reload automatically when Jenkins is ready}
```

Comment:
Why remove the trailing period? The other line is (or was) a title so didn't need it, but this one isn't.

Reply:
Personal preference, I don't personally see the value of them for things like this. Would be a good thing to standardise across Jenkins, as right now many fields include periods, many don't.

Reply:
> many fields include periods, many don't

Are there many _sentences_ that don't have periods?